### ISSUE TITLE
Description fields should include links to where symbols are defined

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1196]
The description field of TMs, GDs, DDs, and IMs currently defines each symbol used in the equation. If a symbol is defined by its own DD, a link to this DD should also appear in the symbol description, like in this example from manual SSP:
![image](https://user-images.githubusercontent.com/18634568/56054870-f8272980-5d25-11e9-945a-5adb26dbe305.png)

Since the Description field is fully automated in Drasil, I can't include such links. I instead have to add them to the Notes field. However, this seems like something that should be possible to automate and then include in the Description field.

**Comment by smiths** [comment]
@bmaclach, I completely agree.  We added the Notes field so that information would not be lost, but we always hoped to replace it with future automations.  An automatically generated link to further details on a data definition would be an improvement.  Following on the work of @halonazhao, this would improve our documentation's automatic traceability.

Assuming @JacquesCarette agrees, we should create an issue for this proposed automation.  Where it sits in our list of priorities is something we should discuss.  That will also depend on how easy/hard this change is to make.

**Comment by Mornix** [comment]
I think #1164 should be able to help with any missing pieces here.

**Comment by JacquesCarette** [comment]
Agreed.

**Comment by smiths** [comment]
@bmaclach, can we close this issue?  It looks to me like the links are now automatically added in the Notes field.

**Comment by bmaclach** [comment]
@smiths No, this still needs doing. The Notes field is defined manually, whereas the Description field is built automatically. We want the links to be included in the Description field so that a user does not need to manually add them.

**Comment by smiths** [comment]
Thank you for the clarification @bmaclach.
